austation/code/modules/admin/verbs/kill_rads.dm

```diff
@@ -17,10 +20,11 @@
 		message_admins("[key_name_admin(usr)] killed rads with range [range] in area [T.loc.name]")
 		log_game("[key_name_admin(usr)] killed rads with range [range] in area [T.loc.name]")
 		for(var/atom/things in range(range,T))
-			if(notradkillable[things.type])
+			if(NOT_RAD_KILLABLE[things.type])
```

Comment:
keep the static variable please, list access on a define is ugly as shit and very confusing.
there was nothing wrong with it in the first place. and there isn't a single instance of defines being used to hold lists of types like this in the codebase.

Reply:
Use type list instead, don't do that lol

Reply:
It was elon that screeched at me to turn the list into a define -_-

Reply:
I told ya nobody does that